src/DependencyInjection/Compiler/RegisterHookListenersPass.php

```diff
@@ -54,6 +54,8 @@ private function getHooks(ContainerBuilder $container): array
             foreach ($tags as $attributes) {
                 $this->addHookCallback($hooks, $serviceId, $attributes);
             }
+
+            $container->findDefinition($serviceId)->setPublic(true);
```

Comment:
Are you aware that `findDefinition()` no longer includes aliases? We therefore check services and aliases separately in the [MakeServicesPublicPass](https://github.com/contao/core-bundle/blob/4.6/src/DependencyInjection/Compiler/MakeServicesPublicPass.php) class.

Reply:
I am not aware of that… But can an alias be tagged and would it show up with the original service name or the alias name in `findTaggedServiceIds`?

Reply:
I don't know.

Reply:
Fixed in a8fbc7f